projects/packages/backup/package.json

```
@@ -26,7 +26,9 @@
 		"extends @wordpress/browserslist-config"
 	],
 	"dependencies": {
+		"@automattic/components": "2.0.1",
 		"@automattic/jetpack-analytics": "workspace:*",
+		"@automattic/format-currency": "1.0.1",
```

Alphabetical order is nice to maintain.
```suggestion
		"@automattic/components": "2.0.1",
		"@automattic/format-currency": "1.0.1",
		"@automattic/jetpack-analytics": "workspace:*",
```

---

Thanks for the suggestion! Just commited it 😊